components/buttons/NewConversationButton.tsx

```diff
@@ -88,7 +81,7 @@ export default function NewConversationButton({
   };
 
   return (
-    <div className='pb-10'>
```

Comment:
I think we need that for mobile view otherwise the button is hidden at the bottom of the screen

Reply:
I checked the display in mobile mode, and everything looks fine. Please see the attached screenshot for reference. The button isn't hidden at the bottom of the screen. Let me know if you still encounter any issues on your device.
![Screenshot from 2024-08-02 12-25-26](https://github.com/user-attachments/assets/f158643d-70c7-4218-a274-c7c10245b892)